UnitedIncome/components
# [Innovation Sprint] Accessibility Changes for Components

### Description
> Please provide a brief description of your changes below.

* Removes `outline: 0` and `outline: none`. These properties now get toggled off using the `:focus-visible` polyfill. In the Storybook preview you can click on a component, but you will not see the focus state. If you click tab on your keyboard you will. Magic! Learn more: https://a11yproject.com/posts/never-remove-css-outlines/

* Fixes some general rules that were flagged.

* Added the `:focus-visible` polyfill to the preview and added a note in the readme file.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Click on the button, you'll see no outline. Tab to the button, you'll see an outline.

### Comments
> Any additional comments that will aid with the review process.

Innovation sprint!
